docker-distribution/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:3.5-jre8-alpine
+MAINTAINER varkockova.a@gmail.com
+
+USER root
+COPY build.gradle /
+WORKDIR /
+ARG PLUGIN_VERSION
+# ARGs are not persisted so we must store value of ARG to ENV
+ENV PLUGIN_VERSION=$PLUGIN_VERSION
```

Comment:
not sure I get it. ENV is something that I provide when I run the image, right? I do not want user of this image to provide version of plugin in docker run...